certmanager/ca_test.go

```diff
@@ -49,3 +67,96 @@ func TestSelfSignedIstioCA(t *testing.T) {
 		t.Errorf("Generated certificate does not contain a SAN field")
 	}
 }
+
+// Pass in unmatched chain and cert to make sure the `verify` method yeilds an error.
+func TestInvalidIstioCAOptions(t *testing.T) {
+	rootCert := `
+-----BEGIN CERTIFICATE-----
```

Comment:
Updated the timestamps to make the certs long-lived so expiration shouldn't be an issue.